articles/020_ros_with_dds.md

```diff
@@ -163,7 +165,7 @@ eProsima's implementation also includes a minimum DDS API, IDL support, and auto
 
 Given the relatively strong LGPL option and the encouraging but custom license from RTI, it seems that depending on and even distributing DDS as a dependency should be straightforward.
 One of the goals of this proposal would be to make ROS 2 DDS vendor agnostic.
-So, just as an example, if the default implementation is Connext, but someone wants to use one of the LGPL options like OpenSplice or FastRTPS, they simply need to recompile the ROS source code with some options flipped and they can use the implementation of their choice.
+So, just as an example, if the default implementation is Connext, but someone wants to use one of the Apache 2 options like Fast DDS or Cyclone DDS, they simply need to recompile the ROS source code with some options flipped and they can use the implementation of their choice.
```

Comment:
While updating the license and changing the vendor names I would suggest to also update the surrounding information. Connext isn't the default - Fast DDS is. "Flipping some options" isn't really helpful without mentioning them - and afaik there shouldn't be any need for options - just rebuilding from source with the desired RMW implementation present should make it usable.